jmix-ui/ui/src/main/java/io/jmix/ui/app/bulk/BulkEditorWindow.java

```diff
@@ -412,7 +413,8 @@ protected FetchPlan createFetchPlan(MetaClass metaClass) {
             String fqn = generateFqn(metaProperty, null);
 
             if (!managedFields.containsKey(fqn)
-                    && !managedEmbeddedProperties.contains(fqn)) {
+                    && !managedEmbeddedProperties.contains(fqn)
+                    && isNotTenantMetaProperty(metaProperty)) {
```

Comment:
For my taste, it's better when methods don't contain negation in themselves (until this is explicitly required). Maybe we should modify this to `!isTenantMetaProperty(...)`?